app/controllers/application_controller/feature_flags_dependency.rb

```diff
@@ -65,7 +69,7 @@ def search_assignments_enabled?
   helper_method :search_assignments_enabled?
 
   def lti_launch_enabled?
-    GitHubClassroom.flipper[:lti_launch].enabled?
```

Comment:
It appears that those tests within `spec/integration/lti_launch_spec.rb` fail because of this change to this feature check. 

It's important that possibly-unauthenticated users still be able to launch into GitHub Classroom from their LMS so we can immediately persist that launch data for the session and _then_ log the user in normally if they are not already, and we definitely want to be able to test that this is indeed what happens (otherwise we have no idea who the session launch data belongs to).

Perhaps we want something like `GitHubClassroom.flipper[:lti_launch].enabled? || (logged_in? && current_user.feature_enabled?(:lti_launch))` to ensure in testing on production we can still enable it just for staff?